src/test/resources/examples/oneOfMarkerInterface/models/Models.kt

```diff
@@ -0,0 +1,27 @@
+package examples.oneOfMarkerInterface.models
+
+import com.fasterxml.jackson.`annotation`.JsonProperty
+import javax.validation.Valid
+import javax.validation.constraints.NotNull
+import kotlin.String
+
+public data class Container(
+    @param:JsonProperty("state")
+    @get:JsonProperty("state")
+    @get:Valid
+    public val state: State? = null,
```

Comment:
Here I am not sure whether I it reference State when the Polymorphic annotations are missing, or whether I should have it set to Any to ensure deserialization is not broken

Reply:
Nor I. Well, the model generated is correct but having a model that fails at deserialization defeats the purpose  🤔  I'd slightly lean for `Any`, but I'm on the fence.

Reply:
OK, changed to Any. That at least preserves the existing behaviour better. If anyone comes looking for it, we can add a new configuration option to allow them to opt-in